app/Models/WifiConnection.php

```diff
@@ -48,4 +49,9 @@ public function getColor()
 
         return 'green';
     }
+
+    public static function notifications()
+    {
+        return User::role(Role::INTERNET_USER)->where('reachedWifiConnectionLimit', true)->count();
```

Comment:
Got `Unknown column 'reachedWifiConnectionLimit' in 'where clause' (SQL: select count(*) as aggregate from `users` where exists (select * from `roles` inner join `role_users` on `roles`.`id` = `role_users`.`role_id` where `users`.`id` = `role_users`.`user_id` and `name` = internet-user) and `reachedWifiConnectionLimit` = 1)` for this line.

Reply:
Weird, worked for me, but not anymore. Let me check...

Reply:
Probably the cache stored the good results, and the query did not run. Now it should work.